Stockchecker.js

```diff
@@ -84,4 +83,4 @@ function doesntexist(element, name) {
 
 
 
-console.log(doesntexist(product, 'spoon'));
\ No newline at end of file
+console.log(doesntexist(product, 'Table'));
```

Comment:
exist function returns true if the product exists, but it returns undefined or doesn't return anything otherwise